TREP-LABS/realdrip-backend
# #168190740 Back-end deployment pipeline

#### What does this PR do?
- Configure TravisCI for Continuous Integration/Delivery
- Improve folder structure

#### Description of Task to be completed?
Setup deployment pipeline for back-end app

#### How should this be manually tested?
- Whenever you create a new branch and push some code changes to Github
- CI Jobs(Unit testing/Static analysis) should be executed for the new code changes
- Whenever any code change is merged to the master branch, the CI Jobs should be executed and the master branch should be deployed to Staging.

#### What are the relevant pivotal tracker stories?
[#168190740](https://www.pivotaltracker.com/story/show/168190740)
